src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/DSA/DSAImportExport.cs

```diff
@@ -137,7 +137,7 @@ public static void ImportRoundTrip(bool includePrivate)
         [InlineData(true)]
         public static void ExportAfterDispose(bool importKey)
         {
-            DSA key = importKey ? DSAFactory.Create(DSATestData.GetDSA1024Params()) : DSAFactory.Create(512);
+            DSA key = importKey ? DSAFactory.Create(DSATestData.GetDSA1024Params()) : DSAFactory.Create(1024);
```

Comment:
I bumped this up to 1024 since using an auto-generated 1024-bit key should still validate the test and Android doesn't support keys smaller than 1024.